check-google-java-format.py

```diff
@@ -22,7 +22,12 @@
 import subprocess
 import sys
 import tempfile
-import urllib
+
+
+try:
+    from urllib import urlretrieve
```

Comment:
Could you add an end-of-line comment `# Python 2` here and `# Python 3` two lines below?  (And the some elsewhere that this is used?